projects/packages/search/src/instant-search/components/overlay.scss

```diff
@@ -61,14 +89,7 @@ $overlay-horizontal-padding-lg: 3em;
 
 // Ensure the hidden overlay doesn't add whitespace to printed pages
 @media print {
-    .jetpack-instant-search__overlay.is-hidden {
-        display: none;
-    }
-}
-
-// Only show animation to users who have not chosen reduced motion
-@media (prefers-reduced-motion: no-preference) {
```

Comment:
This shouldn't have been removed (looks like a rebase issue). It was added recently in https://github.com/Automattic/jetpack/pull/25630.